src/main/java/org/folio/list/domain/ListEntity.java

```diff
@@ -31,7 +33,7 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @Table(name = "list_details")
-public class ListEntity {
+public class ListEntity implements Serializable {
```

Comment:
Do we actually need these classes to be `Serializable`? `Serializable` has some warts and should generally be avoided unless you actually _need_ it

Reply:
Side effect from it getting stored as a field in our exceptions, sonar wanted it serializable. I don't see any reason these entities can't be, though?

Reply:
I always forget most of the details, but _Effective Java_ has an entire chapter about `Serializable` just pretty much boils down to, "don't (unless you really, really, really, _really_, need it; in which case, be very, very, very careful")